Tiqr/eduid-app-android
# TIQR-301 & TIQR-306:

update core to include fixes

### Pull Request Checklist

### Short Description of Change
TIQR-301 & TIQR-306:
update core to include fixes

### Motivation and Context
2 for 1 special

### Testing Steps
After enrolling a user check the domain is shown on the account activated screen.
After authenticating check that the buttons on the login confirmation screen are Allow/Deny

### Additional Information
